widget/playground/src/components/ColorPicker/ColorPicker.styles.ts

```diff
@@ -1,10 +1,13 @@
 import { Button, styled } from '@rango-dev/ui';
 
 export const Container = styled('div', {
+  display: 'flex',
+  justifyContent: 'space-around',
+  alignItems: 'center',
   position: 'relative',
 });
 
-export const Color = styled('div', {
+export const ColorDiv = styled('div', {
```

Comment:
We usually don't use this naming convention. maybe use ColorContainer or something like that.

Reply:
fixed